modules/compute/gke-node-pool/gpu-direct-workload/scripts/enable-tcpx-in-workload.py

```diff
@@ -57,8 +58,10 @@ def main():
         file.write(updated_job)
 
     # Step 7: Provide instructions to the user
-    print("\nA new manifest had been generated and updated to have TCPX enabled based on the provided worklad,")
-    print("It can be found in the same path as the original workload file with name ending \"-tcpx\"")
+    print("\nA new manifest has been generated and updated to have TCPX enabled based on the provided workload.")
+    print("It can be found in {path}".format(path=os.path.abspath(new_file_name)))
+    print("You can use the following commands to submit the sample job:")
```

Comment:
Missing newline, also it would be great if you can paste the output from an actual run in the PR description.

Reply:
print() has newline in place
added sample output in description